### DISCUSSION TITLE
How could i add new item on Nav bar

### DISCUSSION THREAD

**Discussion by blackyzero** [vendurehq/vendure#1620] [Q&A]
Hello,

Just fresh install Vendure with demo data, using Vercel commerce NextJS storefront.
The first question, how could i add new Nav bar menu using Admin UI dashboard?

Thank you.

**Comment by blackyzero** [comment]
It looks like issue with Vercel commerce storefront.
Somehow it doesn't appear new collection.

**Reply by michaelbromley** [reply]
Yes there are aspects of that storefront which are just hard-coded. You would need to make some modifications to the components involved. If you want to use React, our Remix starter is much more fully featured.

**Comment by blackyzero** [comment]
got it. Thanks.
I just did fresh install Remix storefront. It's really fast, still discovering it.
